docs/source/getting-started/intro-quickstart/index.rst

```diff
@@ -60,6 +65,25 @@ MLflow is available on PyPI. If you don't already have it installed on your syst
 
         pip install mlflow
 
+Installing a Release Candidate (RC)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are eager to test out new features and validate that an upcoming release of MLflow will work well in your infrastructure, installing the latest 
+release candidate may be of interest to you. 
+
+.. note:: 
+    
+    Release Candidate builds are not recommended for actual use, rather they are intended only for testing validation. 
+
+To install the latest version of MLflow's release candidates for a given version, see the example below that uses MLflow 2.14.0 as an example:
+
+.. code-section::
+
+    .. code-block:: bash
+        :name: download-mlflow-rc
+
+        pip install --pre mlflow==2.14.0rc*
```

Comment:
```suggestion
        # the latest release candidate
        pip install --pre mlflow

        # specific release candidate
        pip install mlflow==2.14.0rc0
```

Reply:
I don't think we need both commands. The one install a specific releaese candidate should be enough:

```suggestion
        pip install mlflow==2.14.0rc0
```

Reply:
Adjusted the install to make it clear how to install RCs (latest vs specific)